packages/slice-machine/server/src/api/slices/delete.ts

```diff
@@ -149,8 +146,7 @@ export async function deleteSlice(req: {
 
   const updateLibraries = async () => {
     try {
-      resetLibraryIndex(desiredLibrary?.path);
-      await onSaveSlice(env);
+      await reGenerateLibrariesState(env);
```

Comment:
should we update all libraries or the one that is associated to the deleted slice ?

Reply:
I'm not sure. In the rename we update all of them, so I wasn't sure if this was because of some links between them? Otherwise we should probably just update one

Reply:
A library is basically a folder in your file system.
Updating a slice means updating a particular library.
For that, we just need to regenerate one index normally.
I guess it's ancient code that doesn't make sense anymore.

Is it too much work to change ?
What do you think ?

Reply:
Not much to change, have updated it 👍